### ISSUE TITLE
New Elixer Vueify Error

### ISSUE THREAD

**Issue by weblee** [laravel/elixir#348]
Jeff you just released this:  https://github.com/laravel/elixir/releases/tag/4.0

I cant get the dependencies correct. I have event deleted the node_modules folder and have done a fresh install. 

This is my package.json file

```
{
  "private": true,
  "devDependencies": {
    "gulp": "^3.9.0",
    "laravel-elixir": "^4.0.1",
    "laravel-elixir-vueify": "^1.0.0",
    "promise": "^7.0.4",
    "vue": "^1.0.8",
    "vue-resource": "^0.1.17",
    "vueify": "^6.0.0"
  },
  "dependencies": {}
}
```

Error:
```
npm WARN EPEERINVALID vueify@6.0.0 requires a peer of babel-runtime@^6.0.14 but none was installed.
```

Are the dependencies correct versions?

**Comment by JeffreyWay** [comment]
What happens if you try:

```
{
  "private": true,
  "devDependencies": {
    "gulp": "^3.9.0",
    "laravel-elixir": "^4.0.1",
    "laravel-elixir-vueify": "^1.0.0",
    "promise": "^7.0.4",
    "vue-resource": "^0.1.17",
  },
  "dependencies": {}
}
```

**Comment by weblee** [comment]
Hi Jeff,

I actually had that before. But I did remove and uninstall it all again and still same issue.

Lee

**Comment by JeffreyWay** [comment]
I can't reproduce. Just did a fresh install of Laravel, pulled in `laravel-elixir-vueify`, and did `npm install`. All dependencies are there.

**Comment by weblee** [comment]
Jeffery I know you have closed this but I have the same packages you have above and still getting the same error. Not sure if it matters but the is a brand new laptop so all of this is a fresh install??

 The fresh pull of laravel is install version 3:  https://github.com/laravel/laravel/blob/master/package.json

**Comment by JeffreyWay** [comment]
We haven't updated the laravel/laravel repo yet. Are you installing Elixir 4.0 or 3.*?

**Comment by JeffreyWay** [comment]
Also, make sure you're using current versions of Node and NPM. I'm on Node 5.1 and NPM 3.3.12.

**Comment by weblee** [comment]
I changed to version 4.01 which you can see above. And yes same for node and npm..

~ » npm -v                                               lee@Lees-MacBook-Pro-2
3.3.12
------------------------------------------------------------
~ » node -v                                              lee@Lees-MacBook-Pro-2
v5.1.0
------------------------------------------------------------
~ »

Ill move back down to version 3 as I dont want to bother you. But maybe there is a dependency conflist with elixer verion 4?

**Comment by weblee** [comment]
I just cant get anything to install correctly..

I have deleted the node_modules folder. Set my package file to the below and Im still getting errors.

```
{
  "private": true,
  "devDependencies": {
    "gulp": "^3.8.8",
    "laravel-elixir": "^3.0.0"
  },
  "dependencies": {}
}
```

Errors
```
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of vueify-insert-css@^1.0.0 but none was installed.
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of babel-runtime@^5.8.25 but none was installed.
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of vue-hot-reload-api@^1.2.0 but none was installed.
```

Is this just me?

**Comment by Banjerr** [comment]
i'm also getting the exact same errors as @weblee 
this is my package.json. it's a fresh Laravel install, 
```
{
  "private": true,
  "devDependencies": {
    "gulp": "^3.9.0",
  },
  "dependencies": {
    "laravel-elixir": "3.4.2",
    "bootstrap-sass": "^3.0.0"
  }
}
```
my node/npm versions
```
 → npm -v
3.3.12
 → node -v
v5.1.0
```

trying to run sass gives me this error
```
module.js:340
    throw err;
    ^

Error: Cannot find module 'inflight'
```

**Comment by Fuxy22** [comment]
I get the same error as @weblee on a custom docker image based on the ubuntu 14.04 image with nodejs v5.4.0 and npm v3.3.12

```bash
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of vueify-insert-css@^1.0.0 but none was installed.
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of babel-runtime@^5.8.25 but none was installed.
npm WARN EPEERINVALID vueify@5.0.4 requires a peer of vue-hot-reload-api@^1.2.0 but none was installed.
```

https://gist.github.com/Fuxy22/b3c4fec85cad87c6de01 - This is basically what I'm using. 

I apologize in advance for the mess this is my first attempt at getting everything to play nice together.

**Comment by Fuxy22** [comment]
To update everyone I was using Laravel 5.1 and it required laravel-elixir ^3.0.0

I was able to fix the issue while updating to Laravel 5.2 and laravel-elixir: ^4.0.0
